tests/ui/numbers-arithmetic/bitwise-ops-platform.rs

```diff
@@ -1,3 +1,5 @@
+//! Tests bitwise operations with platform-specific and negative number behavior.
```

Comment:
Remark: honestly, this test slightly confuses me because it doesn't seem to be covering all of the interesting boundary cases 🤔 Not for this PR tho.